src/libraries/System.IO.Compression/src/System/IO/Compression/CompressionMode.cs

```diff
@@ -3,9 +3,18 @@
 
 namespace System.IO.Compression
 {
+    /// <summary>Specifies whether to compress or decompress the underlying stream.</summary>
+    /// <remarks>This enumeration is used with the <see cref="System.IO.Compression.GZipStream" /> and <see cref="System.IO.Compression.DeflateStream" /> classes.</remarks>
```

Comment:
We should fix the formatting of these tags before running the automation on everything.

```suggestion
    /// <remarks>This enumeration is used with the <see cref="System.IO.Compression.GZipStream"/> and <see cref="System.IO.Compression.DeflateStream"/> classes.</remarks>
```

Reply:
We've always had the space character in there in dotnet-api-docs.

Reply:
VS does not insert the trailing space before the slash. I've wondered if it would be better to be consistent with what VS is going to do. I've opened https://github.com/carlossanlop/DocsPortingTool/issues/73 for that, but we don't need to block this PR on that issue; we can run a revision run to reformat.